### DISCUSSION TITLE
第四版已经出了，有没有翻译计划呢

### DISCUSSION THREAD

**Discussion by cloudflaresni** [kanition/pbrtbook#136] [问答]
首先感谢译者的辛苦努力，本人也在学习图形学，希望能够在某种程度上做出贡献

**Comment by kanition** [comment]
谢谢支持！这个影响因素较多：
1.本人业余时间是否充足：直接影响进度（最近停更就是这个原因，我其实一直想继续弄的，大半年前第8章就已经过半了，现在却都还没弄完，何况第8章很多公式是从天而降的，明显要补充物理光学材料）
2.本人自学情况：比较难理解的部分会翻译得更慢，必要时会自行编写补充材料，也会延缓进度（目前已有的几个补充小节耗费的精力完全不亚于翻译完整的一章，因为除了翻译还要互相验证查到的各种材料并尽量理顺逻辑组织起来）
3.第4版与第3版的差别：如果修订量不大，可以考虑在第3版基础上参照修订变为继续翻译第4版，否则得另起一版

我十分期待第4版，很想阅读和翻译，不过目前考虑到个人精力有限，感觉希望有点渺茫。目前我还是计划先完成第3版内容。这样了解全书内容后，说不定也能对更好翻译第4版有帮助

**Reply by kanition** [reply]
@HakkerBarry 收到来信，感谢支持，关于第4版的情况如上。
